app.d/prometheus.py

```diff
@@ -146,6 +149,18 @@ def update(app: ApplicationState):
     result_dynamic_average = result_dynamic.dropColumns("DateTime", "Job", "Instance").avgBy("PrometheusQuery")
     app.setField("result_dynamic_average", result_dynamic_average)
 
+    #Downsampling examples
+    result_dynamic_downsampled_average = result_dynamic.update("DateTimeMinute = lowerBin(DateTime, 60000000000)")\
```

Comment:
One way you can clean this up is to use `'00:01:00.123'` type of notation for the bin size.

Reply:
Didn't know that was possible for a parameter. It's not documented on the java/pydocs or the deephaven.io page. I'll open up a ticket to get this documented